requirements-dev.txt

```diff
@@ -180,6 +180,7 @@ sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==2.0.0
 sphinxcontrib-serializinghtml==2.0.0
 sphinxext-opengraph==0.9.1
+sphinxext-rediraffe==0.2.7
```

Comment:
Are we still maintaining the requirements files?

Reply:
We still have a couple of workflows that depend on this file. https://github.com/canonical/charmcraft/pull/2143 gets us most of the way to no longer needing it though.